vbox/Dockerfile

```diff
@@ -89,9 +89,11 @@ VOLUME ["/cuckoo/conf"]
 
 EXPOSE 1337 31337
 
-ADD --chown=cuckoo:cuckoo https://raw.githubusercontent.com/$CUCKOO_FORK_REPO/cuckoo/machinery/virtualbox_websrv.py $CUCKOO_PACKAGE_PATH/machinery/virtualbox_websrv.py
-ADD --chown=cuckoo:cuckoo https://raw.githubusercontent.com/$CUCKOO_FORK_REPO/cuckoo/common/config.py $CUCKOO_PACKAGE_PATH/common/config.py
```

Comment:
did you remove the chown here because dockerCloud doesn't support the flag?

Reply:
I removed it mainly because `--chown=cuckoo:cuckoo` sets 0600 permissions on these files which gives another hassle later when I change user's uid in entrypoint.